cmd/operator/main.go

```diff
@@ -54,6 +54,33 @@ const (
 	logFormatJson   = "json"
 )
 
+type namespaces map[string]struct{}
+
+// Set implements the flagset.Value interface.
```

Comment:
very small nit, instead of putting this into godoc, you could
```
var _ flag.Value = (namespaces)(nil)
```
to let the compiler check that `namespaces` implements `flag.Value`

Reply:
sounds good, agreed :+1: